### PR TITLE
Improving Website

### PR DESCRIPTION
**Pull request by KommToby**
General cleanup and added a few points of internal commentary to support other people helping out with making the website. Hopefully some of the issues i've had with the imaging can be resolved by other people or when I learn enough about backgrounds to do it myself.

- Added a subtitle to the title of the webpage referencing the NSE placements of this year.
- Cleaned up the HTML code to all direct to the css file instead of styling within the HTML itself
- Added a NSE.png to reference when showing the page

### CODE REVIEW

**Comment by KommToby** [comment]
Been working a lot on the website, so i'd recommend having a brief view of all of the previous commits before looking at the final one before accepting it through.

I have an issue with the fact I want to change the main html file to home.html but I don't know how the domain we have works with the website, It seems like you originally set it up for all of the websites to be homed within one html file, but that's really inefficient and messy

So I was hoping there was a way for us to make the files orientated like
wattesports.co.uk/home
wattesports.co.uk/contact
etc
Instead of having everything under wattesports.co.uk (Which I assume is linked to the index.html file) and linking all of the other "pages" to other hidden content on the page

File: About.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<meta name="msapplication-TileColor" content="#da532c">
+<meta name="theme-color" content="#ffffff">
+
+
+<link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+<link href="index.css" rel="stylesheet">
+
+
+</head>
+<body> <! Can we add a transparent background of our logo? I've tried this but I couldn't seem to get the scaling correct>
```
Comment (Danial-Q) [inline About.html]: `opacity`, `z-index` and just general sizing css properties should achieve this

File: About.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	<header>
+		<div class="topnav"> 
+			<a href="index.html">Home</a>
+			<a href="news.html">News</a>
+			<a href="contact.html">Contact</a>
+			<a class="active" href="about.html">About</a>
+			<a class="right" href="https://discord.wattesports.co.uk">Discord</a>
+		</div>
+		<div>
+			<a href=https://www.hw.ac.uk/> <img src="hw-logo.svg" alt="Heriot Watt Logo" width="170" style="float: right;padding: 20px 20px 20px 20px;"> </a>
```
Comment (Danial-Q) [inline About.html]: Should consider making a folder for all images and importing from there just to keep stuff organized. Same goes for every new page added and such

File: Contact.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+<link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+<link href="index.css" rel="stylesheet">
+
+
+</head>
+<body> <! Can we add a transparent background of our logo? I've tried this but I couldn't seem to get the scaling correct>
+	<header>
+		<div class="topnav">
```
Comment (Danial-Q) [inline Contact.html]: Could use something like react or something to make each page more modular I.E you'd write a `<NavBar />` component then can import that to each page, as to not replicate code between pages. Makes maintenance much simpler too as you would only have to update one place if you decided to add another tile into the NavBar.

Mind react is a JS library, so y'all would to have to learn JS, but I'm more than willing to teach y'all if you want and it'd be a good skill to have.

Same thing could be achieved using `<iframe />`  too which is simpler but has it's risks associated with it

File: Contact.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+	<header>
+		<div class="topnav"> 
+			<a href="index.html">Home</a>
+			<a href="news.html">News</a>
+			<a class="active" href="contact.html">Contact</a>
+			<a href="about.html">About</a>
+			<a class="right" href="https://discord.wattesports.co.uk">Discord</a>
+		</div>
+		<div>
+			<a href=https://www.hw.ac.uk/> <img src="hw-logo.svg" alt="Heriot Watt Logo" width="170" style="float: right;padding: 20px 20px 20px 20px;"> </a>
```
Comment (Danial-Q) [inline Contact.html]: inline styling reeee

File: Contact.html (hunk 3 above)
Comment (KommToby) [reply Contact.html]: I did not do topnav at all, all of that system was In here before I started working on it, I think Kyle or Colin made the base website using some google tool, but i've used iframe before so maybe I can look into this.

File: About.html (hunk 1 above)
Comment (KommToby) [reply About.html]: Could you or someone else give it a bash? Using the background style of the body it seemed limited on what I could do it it (i.e. adding opacity) so I had to do that manually in photoshop, and then once that was imported, I couldn't get the logo to scale with the website correctly when the window was resized.

File: About.html (hunk 2 above)
Comment (KommToby) [reply About.html]: So thinking just files called "images" and "pages" or something?

Can they still be called by just using src="filename.png" or do you have to reference the folder they're stored in too?

Comment (Danial-Q) [reply About.html]: You'd have to reference the folder too but you can just use `..` to go up a level in the file tree so it's fairly simple to do

File: index.css [hunk 5]
```diff
@@ -52,6 +50,38 @@ p{
 	float: right;
  }
 
+/* Class for adding a background image - Not being used right now as I can't get it to work correctly */
+.background{
+	background: url(https://cdn.discordapp.com/attachments/698264458996351016/706052590294663238/Wattesportspartial.png);
```
Comment (Danial-Q) [inline index.css]: Should have this as an img in the repo rather than a link that could eventually be removed

File: News.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+			<a class="active" href="news.html">News</a>
+			<a href="contact.html">Contact</a>
+			<a href="Teams.html">Teams</a>
+			<a class="right" href="https://discord.wattesports.co.uk">Discord</a>
+		</div>
+		<div>
+			<a href=https://www.hw.ac.uk/> <img src="logos/hw-logo.svg" alt="Heriot Watt Logo" width="170" style="float: right;padding: 20px 20px 20px 20px;"> </a>
+		</div>
+			<div class="title">
+			<h1> News </h1>
```
Comment (Danial-Q) [inline News.html]: Adding spaces here will add spaces to the title, it should be `<h1>News</h1>`

File: Teams.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+			<a href=https://www.hw.ac.uk/> <img src="logos/hw-logo.svg" alt="Heriot Watt Logo" width="170" style="float: right;padding: 20px 20px 20px 20px;"> </a>
+		</div>
+			<div class="title">
+			<h1> Teams </h1>
+		</div>
+
+	</header>
+	<main>
+		<div class="main">
+			<p> We currently field 11 teams across League of Legends, CS:GO, Overwatch, Rainbow Six: Siege, DOTA 2, and PUBGM, as well as a selection of solo players in Tekken, SSBU and FIFA.</p>
```
Comment (Danial-Q) [inline Teams.html]: Spaces

File: Contact.html (hunk 4 above)
Comment (Simber1) [reply Contact.html]: I was having issues with the SVG so I inlined it.

**Review by Simber1 — APPROVED** [review]
All looks good 👍

**Pull request merged**